webapp/templates/service-worker.js

```diff
@@ -23,16 +25,20 @@ self.addEventListener(
           if (r) {
             return r;
           }
-          return fetch(e.request)
+
+          // IMPORTANT: Clone the request to reuse in fetch.
+          const request = e.request.clone();
+          const url = new URL(e.request.url);
+          const path = url.pathname;
```

Comment:
This is the part that was broken. `request.url` is just a string and does not have a method called `path`.